test-integration/mock-backend-server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+#FROM adoptopenjdk/maven-openjdk8
+FROM adoptopenjdk:11.0.8_10-jre-hotspot-bionic
+
+#COPY src /home/app/src
+#COPY pom.xml /home/app
```

Comment:
Lets use the argument to get the jar name

Reply:
Fixed in ee03466e7304d4b68a1c39906fb6a12affb17948